src/gmres_e.m

```diff
@@ -0,0 +1,426 @@
+function [x, flag, relresvec, time] = ...
```

Comment:
What about `mvec`? This we have for both `pd_gmres` and `lgmres`. Why we don't provide it for this algorithm?

@jccf19 : Do you any comments about this?

Reply:
Ohh. @jhabriel: I see that the `mvec` in `lgmres` you are mentioning is just an outdated slice of docstring that came from `pd_gmres` and is now obsolete, since both `lgmres(m, k)` and `gmres_e(m, k)` function in a Krylov subspace of constant size $s=m+k$ at any outer iteration, therefore `mvec` is rather unnecesary. In my opinion, we should remove it from `lgmres`.